starport-core/src/main/scala/com/krux/starport/util/notification/EmailNotification.scala

```diff
@@ -0,0 +1,17 @@
+package com.krux.starport.util.notification
+import com.krux.starport.config.StarportSettings
+import com.krux.starport.db.record.Pipeline
+
+protected class EmailNotification extends Notification {
```

Comment:
this can be `object` instead of `protected class`

Reply:
Agreed